src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/magicnumber/InputMagicNumberAnnotationElement.java

```diff
@@ -0,0 +1,11 @@
+package com.puppycrawl.tools.checkstyle.checks.coding.magicnumber;
+
+/**
+ * ignoreAnnotationElementDefaults = true/false
```

Comment:
Andrew, let me explain how this should work: 
One configuration == one test == one input file.

There should be simple
```
/**
 * ignoreAnnotationElementDefaults = true
 */
@interface InputMagicNumberAnnotationElement {
	int value() default 10; // OK
}
```
But the test `testDefault` should be extended to cover
```
@interface InputMagicNumberAnnotationElement {
	int value() default 10; // violation
}
```
if it does not have this case already.

Reply:
I thought about it, but decided to make a separate file for this option.
There is a one big file that is used for all tests, and I didn't want to add more examples there and correct all other test cases. It is difficult to maintain it, as for me.

Reply:
Corrected.

Reply:
@pbludov If you are satisfied with the fix, please mark this conversation resolved.